lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -549,6 +552,82 @@ def upload_csv_to_report_store(rows, csv_name, course_id, timestamp):
     )
 
 
+def is_eligible_for_certificate(embargoed, whitelisted, grade):
+    """
+    Check if user is eligible for certificate.
+    """
+    return not embargoed and (whitelisted or grade is not None)
+
+
+def is_certificate_delivered(embargoed, whitelisted, certificate_status):
+    """
+    Check if certificate is generated and delivered to user.
+    """
+    return not embargoed and (whitelisted or certificate_status['status'] == CertificateStatuses.downloadable)
+
+
+def verification_status(user, course_id):
+    """
+    Returns the verification status.
+    """
+    user_is_verified = SoftwareSecurePhotoVerification.user_is_verified(user)
+    user_is_re_verified = SoftwareSecurePhotoVerification.user_is_reverified_for_all(course_id, user)
+
+    if not user_is_re_verified:
+        return 'ID Verification Expired'
```

Comment:
If the user did not even do the initial verification, by making this the first check wouldn't the status be "ID Verification Expired" instead of "Not ID Verified"?